### PR TITLE
Port compiler requirements to 2021.4

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Add compiler requirements

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
Please, sync with https://github.com/openvinotoolkit/openvino/pull/11159

**Comment by ilyachur** [comment]
> Please, sync with #11159

Done

File: docs/install_guides/installing-openvino-linux.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ Optimized for these processors:
    - [PyPi package](https://pypi.org/project/openvino/)
    - [Docker image](https://catalog.redhat.com/software/containers/intel/openvino-runtime/606ff4d7ecb5241699188fb3)
 
+Software
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-linux.md:51]: ** ?

Comment (ilyachur) [reply docs/install_guides/installing-openvino-linux.md:51]: Fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**